src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -594,9 +607,19 @@ export default {
           }
           filing = res.data.filing
           this.haveChanges = false
-        }).catch(error => {
+        }).catch(async error => {
           if (error && error.response && error.response.status === PAYMENT_REQUIRED) {
-            this.paymentErrorDialog = true
+            const errCode = this.getErrorCode(error)
```

Comment:
Since you're not awaiting getErrorCode(), this catch arrow function doesn't need to be async.

Reply:
the async is necessary if there is an error code, since getErrorOBj is awaited (if it exists), is it not? (line 611)

Reply:
Oops, I missed the getErrorObj() call a couple of lines later. You are correct! I'll resolve this comment now :]